.github/workflows/prod-sync-ami.yml

```diff
@@ -38,9 +38,13 @@ jobs:
             aws configure set region $region_name
             aws configure list | grep region | awk '{print $2}'
             
-            echo "Making AMI Public in region (${region_name}): Open Distro for Elasticsearch-${{ env.od_version }}"
-            AMI_IMAGE_ID=`aws ec2 describe-images --filters "Name=name,Values=Open Distro for Elasticsearch-${{ env.od_version }}*" --query 'Images[*].{ID:ImageId}' --output text`
-            aws ec2 modify-image-attribute --image-id "${AMI_IMAGE_ID}" --launch-permission "Add=[{Group=all}]"
+            echo "Making AMI Public in region (${region_name}): Open Distro for Elasticsearch-${{ env.od_version }}-x86_64"
+            AMI_IMAGE_ID_x64=`aws ec2 describe-images --filters "Name=name,Values=Open Distro for Elasticsearch-${{ env.od_version }}-x86_64*" --query 'Images[*].{ID:ImageId}' --output text`
+            aws ec2 modify-image-attribute --image-id "${AMI_IMAGE_ID_x64}" --launch-permission "Add=[{Group=all}]"
+            
+            echo "Making AMI Public in region (${region_name}): Open Distro for Elasticsearch-${{ env.od_version }}-arm64"
```

Comment:
Can you modify echo command to include the architecture too? It will be confusing which AMI it is making public

Reply:
I didnt get you, the echo already has the architecture "Open Distro for Elasticsearch-${{ env.od_version }}-arm64"

Reply:
Ahh it went on next line so missed it.